ofl/anektelugu/METADATA.pb

```diff
@@ -0,0 +1,169 @@
+name: "Anek Telugu"
+designer: "Ek Type"
+license: "OFL"
+category: "SANS_SERIF"
+date_added: "2022-02-09"
+fonts {
+  name: "Anek Telugu"
+  style: "normal"
+  weight: 400
+  filename: "AnekTelugu[wdth,wght].ttf"
+  post_script_name: "AnekTelugu-Medium"
+  full_name: "Anek Telugu Medium"
+  copyright: "Copyright 2021 The Anek Project Authors (https://github.com/EkType/Anek)"
+}
+subsets: "latin"
+subsets: "latin-ext"
+subsets: "menu"
+subsets: "telugu"
+axes {
+  tag: "wdth"
+  min_value: 75.0
+  max_value: 125.0
+}
+axes {
+  tag: "wght"
+  min_value: 100.0
+  max_value: 800.0
+}
+languages: "aa_Latn"
```

Comment:
Which version of gftools are you using?

Each of these language fields should have a comment.

Reply:
I will check. Please halt all Anek reviewing anyways. I'll submit PR amendments later today for different reasons

Reply:
I checked: I was using the latest github version of gftools. I confirmed with the latest pypi version, too. Both have no language comment